docs/theory/diffusion/diffusion.rst

```diff
@@ -0,0 +1,667 @@
+Diffusion analysis
+==================
+
+.. only:: html
+
+    :nbexport:`Download this page as a Jupyter notebook <self>`
+
+Pylake supports a number of methods for studying diffusive processes.
+These methods rely on quantifying the movement of the fluorophore.
+This section details some of the available methods and details some pitfalls with each of them.
+In this chapter, we will be making use of simulated data, so that we can study the quality of our estimates while knowing the ground truth.
+Note that the code to perform these simulation is included, so that you can play with them and produce similar graphs which are more tailored to your own experimental conditions.
+For more information on tracking the particles in a kymograph, refer to :doc:`Kymotracking </tutorial/kymotracking>` and specifically :ref:`the section on diffusion <kymotracking-diffusion>` for specifics regarding diffusion analysis on kymographs.
+
+Simulating 1D free diffusion
+----------------------------
+
+The displacements of a particle undergoing free diffusion are uncorrelated over time and distributed according to a normal distribution.
+The standard deviation of this distribution is given by Einstein's equation.
```

Comment:
The standard deviation of this distribution follows from the diffusion law, $\sigma = \sqrt{2 \Delta} t$.

The Einstein relation, which relates the diffusion constant to the Boltzman's constant and viscosity, does not give the standard deviation?

Reply:
Oh whoops. I meant to write follows from Einstein's equations for Brownian motion. I was referring to Eq 11 in this: http://users.physik.fu-berlin.de/~kleinert/files/eins_brownian.pdf

I think your equation is missing a `D`? 😅 

Would you be ok with the following?:

The standard deviation of this distribution follows from the theory of Brownian motion [cite]: $\sqrt{2 D \Delta t}$?

Or even just:

The standard deviation of this distribution follows from the diffusion equation: $\sqrt{2 D \Delta t}$?

I guess it's established enough that it doesn't require a specific citation. I'm also not sure it's usually referred to as a law?

Reply:
ah yes, Einstein wrote many equations 😅 
I prefer calling it 'diffusion equation'. A citation is indeed not necessary.

Reply:
That he did, that he did :D

I will amend it!